fuels-abigen-macro/tests/test_projects/complex_types_contract/Forc.toml

```
@@ -0,0 +1,9 @@
+[project]
+author  = "Rodrigo Araujo"
+license = "MIT"
```

In a separate PR, can you make this change to current tests?
```suggestion
author  = "Fuel Labs <contact@fuel.sh>"
license = "Apache-2.0"
```